CHANGELOG.md

```diff
@@ -21,10 +21,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#89](https://github.com/cyberark/conjur-api-python3/issues/89)
 
 ### Changed
+- The CLI and SDK now uses a system's native credential store to save credentials instead of a netrc file. If a store
+  is not available, the credentials will be saved to the netrc as a fallback.
```

Comment:
how does this affect the user? `uses a system's native credential store to save credentials instead of a netrc file` looks like an implementation detail

Reply:
it changes their configurations and the default behavior of the CLI. I think it is important to write because we are changing the default behavior. I don't want users to be looking for netrc for their creds when they login and think something might be wrong if they don't see them